src/styles/pages/home/home.module.scss

```diff
@@ -0,0 +1,182 @@
+// Mission statement and paragraph
+.mission {
+    display: flex;
+    align-items: center;
+    gap: 32px;
+    
```

Comment:
I suggest adding a `column-gap` or `justify-content` property here. The mission section looks good on a laptop screen, but not on bigger devices (I'm viewing on a monitor, so the content is pushed to the left). From quick testing, `space-around` seems to work good, but I'll have you decide.

Reply:
So I tweaked with it a couple of different ways and it seems like just having a bigger textbox width with a column gap between the text and image looks the nicest.

However, it strays a bit from how the figma looks. The figma has the image cropped but the div around the image is still the dimensions of the original image so if I were to replicate it strictly I could add some odd padding on each side. Also, the text seems to be more squared up in the figma.

I tried making it a bit more natural with a wider text width but since it's different than the Figma let me know if I should change it back.